src/modules/nuttx/iotjs_module_stm32f7nucleo-nuttx.c

```diff
@@ -21,6 +21,27 @@
 #include "iotjs_def.h"
 #include "modules/iotjs_module_stm32f7nucleo.h"
 
+#if ENABLE_MODULE_ADC
+static void iotjs_pin_initialize_adc(jerry_value_t jobj) {
+  unsigned int number_bit;
+
+// ADC pin name is "ADC.(number)_(timer)".
```

Comment:
Is this comment ok? Based on the property set below, the name is `ADC<number>_<timer>`, there is no `.` in the name.

Reply:
Good catch, I will fix this in discovery file too